libs/cart/src/drivers/interfaces/transformers/cart-address-transformer.interface.ts

```diff
@@ -0,0 +1,17 @@
+import { InjectionToken } from '@angular/core';
+import { DaffCartAddress } from '../../../models/cart-address';
+
+/**
+ * An interface that must be implemented by address transformer services; e.g. a service that transforms a magento address into a DaffCartAddress.
+ */
+export interface DaffCartAddressTransformerInterface<T extends DaffCartAddress> {
+  /**
+   * Transform a single address into a kind of DaffCartAddress.
+   */
+  transform(address: any): T;
+}
```

Comment:
I would think you'll need a transformIn and transformOut for this, since you'll probably need to go back and forth with this one.